plugins/k8saudit/rules/k8s_audit_rules.yaml

```diff
@@ -14,13 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-- required_engine_version: 11
+
+- required_engine_version: 12
```

Comment:
IIRC this is needed because the legacy k8s audit log fields were removed from the Falco Engine in this version.